packages/upset/src/utils/styles.ts

```diff
@@ -43,4 +43,6 @@ export const mousePointer = css`
 
 export const arrowIconCSS = {
   height: '16px', width: '16px', marginLeft: '6px',
-};
\ No newline at end of file
+};
+
+export const ATTRIBUTE_DEFAULT_COLOR = '#d3d3d3';
```

Comment:
Is this not already accounted for with the grey for the bars?

Reply:
Vega doesn't know what fill color to use for this, so I believe it will default to black if nothing is provided.